From 6cdb0bcf949a532cbdfcf1e92909c5e7dec34d63 Mon Sep 17 00:00:00 2001
From: Daniel Bast <2790401+dbast@users.noreply.github.com>
Date: Sat, 9 Nov 2024 19:45:26 +0100
Subject: [PATCH] Allow setting VERSION_METADATA + GIT_DIRTY

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index fe39841..fe55baa 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ SHELL      = /usr/bin/env bash
 GIT_COMMIT = $(shell git rev-parse HEAD)
 GIT_SHA    = $(shell git rev-parse --short HEAD)
 GIT_TAG    = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
-GIT_DIRTY  = $(shell test -n "`git status --porcelain`" && echo "dirty" || echo "clean")
+GIT_DIRTY  ?= $(shell test -n "`git status --porcelain`" && echo "dirty" || echo "clean")
 
 ifdef VERSION
 	BINARY_VERSION = $(VERSION)
@@ -47,7 +47,7 @@ ifneq ($(BINARY_VERSION),)
 	LDFLAGS += -X helm.sh/helm/v3/internal/version.version=${BINARY_VERSION}
 endif
 
-VERSION_METADATA = unreleased
+VERSION_METADATA ?= unreleased
 # Clear the "unreleased" string in BuildMetadata
 ifneq ($(GIT_TAG),)
 	VERSION_METADATA =
-- 
2.47.0

