From 67b21c2e33726945bcb3712824bc59c248892dca Mon Sep 17 00:00:00 2001
From: Daniel Bast <2790401+dbast@users.noreply.github.com>
Date: Sat, 9 Nov 2024 10:49:16 +0100
Subject: [PATCH] Allow setting VERSION_METADATA

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fe39841..eee5f92 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ ifneq ($(BINARY_VERSION),)
 	LDFLAGS += -X helm.sh/helm/v3/internal/version.version=${BINARY_VERSION}
 endif
 
-VERSION_METADATA = unreleased
+VERSION_METADATA ?= unreleased
 # Clear the "unreleased" string in BuildMetadata
 ifneq ($(GIT_TAG),)
 	VERSION_METADATA =
-- 
2.47.0

